packages/cron-jobs/src/jobs/integrations/media-organizer.ts

```diff
@@ -41,14 +41,24 @@ export const mediaOrganizerJob = createCronJob("mediaOrganizer", EVERY_MINUTE).w
       const start = dayjs().subtract(Number(options.filterPastMonths), "months").toDate();
       const end = dayjs().add(Number(options.filterFutureMonths), "months").toDate();
 
+      const decryptedSecrets = integration.integration.secrets.map((secret) => ({
+        ...secret,
+        value: decryptSecret(secret.value),
+      }));
+
       const sonarr = new SonarrIntegration({
         ...integration.integration,
-        decryptedSecrets: integration.integration.secrets.map((secret) => ({
-          ...secret,
-          value: decryptSecret(secret.value),
-        })),
+        decryptedSecrets,
+      });
+      const radarr = new RadarrIntegration({
+        ...integration.integration,
+        decryptedSecrets,
       });
```

Comment:
Instead you can use the `integrationCreatorByKind` and then pass the kinds in there (specify them for now ` as "radarr" | "sonarr"`). Then just run getCalendarEvents for that one integration and use the events of that